exercise10.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+def get_price_quantity():
+    """Get price and quantity from user"""
+
+    price = float(input("Enter the price:$"))
+    quantity = int(input("Enter the quantity: "))
+
+    return price, quantity
+
+
+def get_subtotal(price, quantity):
+    """Calculate subtotal: price * quantity"""
+
+    return price * quantity
+
+
+def calculate_sum_of_subtotals(subtotal_item1, subtotal_item2, subtotal_item3):
```

Comment:
I'd recommend allowing this function to take in any number of arguments by using the splat operator: `*subtotals`. Then it's more flexible.